webapp-django/crashstats/crashstats/views.py

```diff
@@ -69,6 +69,10 @@
 
 
 def make_correlations_count_cache_key(product, version, platform, signature):
+    assert product
+    assert version
+    assert platform
+    assert signature
```

Comment:
I don't really like using ``assert`` to much in the code. You already make sure all of those have a value before calling the function, what's the point of verifying that twice?